packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -904,7 +904,7 @@ export function getResolver<TSource, TContext, TArgs>({
               let arraySaneData = saneData
 
               if ('limit' in args) {
```

Comment:
All of these `extends object` changes come from this line. Otherwise, TS complains that I cannot access `limit` in `args`.

Reply:
I am not sure if this is the best approach. Please let me know if anyone else has any other opinions.

Reply:
This seems OK to me, also good to have TArgs a it more restrictive now.

I'm not sure about this though, as limit could potentially be a string (or other type). Are the values sanitized?
```
const limit = args['limit'] as number
```

Reply:
This seems OK to me, also good to have TArgs a it more restrictive now.

I'm not sure about this though, as limit could potentially be a string (or other type). Are the values sanitized?
```
const limit = args['limit'] as number
```

Reply:
Good point, I added a type check